www/templates/main-metrics.html

```diff
@@ -0,0 +1,60 @@
+<ion-view view-title="Metrics">
+  <ion-nav-bar class="bar-stable">
+    <ion-nav-back-button></ion-nav-back-button>
+    <ion-nav-buttons side="right">
+      <button class="button button-assertive button-block"
+              ng-click="getMetrics()">Get!</button>
```

Comment:
Can we move the Get button on the content instead of nav-bar the lower half of the button is floating below the navbar.